src/App.module.css

```diff
@@ -1,22 +1,24 @@
 .header {
   grid-area: header;
+  height: 4rem;
 }
 
 .sidebar {
   grid-area: sidebar;
   background-color: #1c59a7;
+  height: 96rem;
 }
 
 .main {
   grid-area: main;
-  background-color: gray;
+  height: 96rem;
```

Comment:
Why 96rem exactly? How about 100vh - header height?

Reply:
no need for this one actually - same reason as the header one above. The parent style (.container) works fine.